build/Dockerfile

```diff
@@ -1,12 +1,7 @@
-FROM centos:7
+FROM fedora:30
 
-RUN yum -y install epel-release && \
-    yum -y update && \
-    yum -y install \
-        https://kojipkgs.fedoraproject.org/packages/nmstate/0.0.8/1.el7/noarch/python2-libnmstate-0.0.8-1.el7.noarch.rpm \
-        https://kojipkgs.fedoraproject.org/packages/nmstate/0.0.8/1.el7/noarch/nmstate-0.0.8-1.el7.noarch.rpm \
-        iproute && \
-    yum clean all
+RUN sudo dnf install -y nmstate iproute && \
```

Comment:
We take whatever is released in repositories our OS provides. Pinning version is something you see with Python and PyPI where multiple versions are available at the same time, but that is not the case here. (pushing back just because of I'm not used to see version request in `dnf`)